AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Support/Constants/AuthPluginErrorConstants.swift

```diff
@@ -59,6 +59,10 @@ struct AuthPluginErrorConstants {
         "There is no user signed in to retreive user sub",
         "Call Auth.signIn to sign in a user and then call Auth.fetchSession")
 
+    static let deleteUserSignOutError: AuthPluginErrorString = (
+        "There is no user signed in to delete an account",
+        "Call Auth.signIn to sign in a user and then call Auth.deleteUser")
```

Comment:
Is it necessary to call `fetchSession` first, or is `signIn` sufficient?

Reply:
signIn is sufficient for deleteUser.  I believe the only time fetchAuthSession is necessary (other than to check signIn status) is to access the underlying credentials directly.